Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Program.cs

```diff
@@ -24,6 +28,10 @@ static void Main(string[] args)
 
 var host = new HostBuilder()
     .ConfigureFunctionsWorkerDefaults()
+    .ConfigureServices(services => {
+        services.AddApplicationInsightsTelemetryWorkerService();
```

Comment:
Don't we need to also add the using statements so that these extension methods are available here?

```csharp
using Microsoft.Azure.Functions.Worker;
using Microsoft.Extensions.DependencyInjection;
```

Reply:
Yes - adding. Just locally tested as well.